Formula 1/ViewModels/CircuitsViewModel.swift

```diff
@@ -0,0 +1,105 @@
+//
```

Comment:
I don't like it at all that you have a bunch of different Types in the same CircuitsViewModel.swift. You should create a new file for each model.

Reply:
We had a discussion where you mentioned that objects that will be used for decoding should not go into a model folder, because i won't ever make a reference of them myself. Where does this belong?

Reply:
There are no that though, these your app models, they belong in a Models folder with each one their own file.